src/modules/tests/test-data/cat-d/pcv-door-exercise/__tests__/pcv-door-exercise.reducer.spec.ts

```diff
@@ -0,0 +1,74 @@
+// import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
```

Comment:
Can we uncomment this or delete it?

Reply:
we will be addressing the tests in the next ticket, which will be analytics :)